src/__configuration__/themes/independence-day.ts

```diff
@@ -0,0 +1,69 @@
+import * as PXBThemes from '@pxblue/react-themes';
+import * as Colors from '@pxblue/colors';
+import * as BrandingColors from '@pxblue/colors-branding';
+import IndependenceDay from '../../app/assets/themes/independence-day/independence-day-bg.png';
+import IndependenceDayCursor from '../../app/assets/themes/independence-day/flag-cursor.png';
+import AppBarTile from '../../app/assets/themes/independence-day/independence-day-appbar-tile.png';
+import { Schedule } from './types';
+
+export const IndependenceDaySchedule: Schedule = {
+    start: new Date(0, 6, 2), // July 2
+    end: new Date(0, 6, 8), // July 7
+    config: {
+        theme: {
+            ...PXBThemes.blue,
+            palette: {
+                ...PXBThemes.blue.palette,
+                type: 'light',
+                primary: {
+                    light: BrandingColors.blue[300],
+                    main: BrandingColors.blue[500],
+                    dark: BrandingColors.blue[700],
+                },
+                secondary: {
+                    light: Colors.red[300],
+                    main: Colors.red[500],
+                    dark: Colors.red[900],
+                },
+            },
+            overrides: {
+                ...PXBThemes.blue.overrides,
+                MuiAppBar: {
+                    ...PXBThemes.blue.overrides?.MuiAppBar,
+                    colorSecondary: {
+                        color: Colors.white[50],
+                        backgroundColor: BrandingColors.blue[500],
+                        '& .MuiInputBase-root': {
+                            color: Colors.white[50],
+                        },
+                        '& .MuiSelect-icon': {
+                            color: Colors.white[50],
+                        },
+                    },
+                },
+                MuiButton: {
+                    ...PXBThemes.blue.overrides?.MuiButton,
+                    outlined: {},
+                    outlinedPrimary: {},
+                },
+            },
+        },
+        drawerActiveBackgroundFade: 0.85,
+        landingPageBanner: {
+            backgroundImage: `url(${IndependenceDay})`,
+            backgroundSize: 'cover',
+            backgroundRepeat: 'no-repeat',
+            backgroundPosition: 'center 25%',
+            backgroundColor: BrandingColors.blue[700],
+            textShadow: `0 0 5px black`,
+            cursor: `url("${IndependenceDayCursor}") 0 0, auto`,
+        },
+        className: 'independence-day',
+        appBarBackground: {
+            backgroundImage: `url(${AppBarTile})`,
+            backgroundSize: '600px',
+            backgroundPosition: 'left 25%',
+        },
+        landingPageTagline: 'Happy Independence Day!',
```

Comment:
You didn't want to go with `'Merica`? lol

Reply:
'Murica!

Reply:
![image](https://user-images.githubusercontent.com/8997218/123953594-63945a80-d975-11eb-855e-3540b6059ae1.png)
![image](https://user-images.githubusercontent.com/8997218/123953688-83c41980-d975-11eb-9aad-53639cc4b5e7.png)

Reply:
I didn't want to brag too much since we're international. It's not too late to update.